tests/flytekit/unit/core/test_type_engine.py

```diff
@@ -968,6 +968,7 @@ class TestFileStruct(DataClassJsonMixin):
         h_prime: typing.Optional[FlyteFile] = None
         i: typing.Optional[A] = None
         i_prime: typing.Optional[A] = field(default_factory=lambda: A(a=99))
+        j: typing.Union[int, FlyteFile] = None
```

Comment:
Should `None` be a part of this Union?

Reply:
It should not be. Sorry for not noticing that. Thanks~